content/en/platform/corda/5.1/reference/corda-cli/topic.md

```diff
@@ -21,26 +21,38 @@ table th:nth-of-type(2) {
-| connect                                 | Connects to the specified server; see [Topic Creation by Direct Connection]({{< relref "../../deploying-operating/deployment/deploying/manual-bootstrapping.md#topic-creation-by-direct-connection" >}}) |
-| -f                                      | The name of the script file genrated; see [Topic Creation by Scripting]({{< relref "../../deploying-operating/deployment/deploying/manual-bootstrapping.md#topic-creation-by-scripting" >}}).            |
-| -c                                      | The number of topics to create in parallel; see [Topic Creation by Scripting]({{< relref "../../deploying-operating/deployment/deploying/manual-bootstrapping.md#topic-creation-by-scripting" >}}).      |
-| create                                  | Creates Kafka topics; see [create](#create).                                                                                                                      |
-| delete                                  | Deletes Kafka topics; see [delete](#delete).                                                                                                                      |
+| Argument                 | Description                                               |
+| ------------------------ | --------------------------------------------------------- |
+| -b, \-\-bootstrap-server | The address of the bootstrap server.                      |
+| -k, \-\-kafka-config     | The path to the Kafka configuration file.                 |
+| -n, \-\-name-prefix      | The name prefix for topics.                               |
+| create                   | Creates the required Kafka topics; see [create](#create). |
 
 ## create
 
+The following table lists the `create` sub-commands and arguments:
+
+| Argument           | Description                                                                                                                                                                          |
+| ------------------ | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| -r, \-\-replicas   | The number of replicas.                                                                                                                                                              |
+| -p, \-\-partitions | The number of partitions.                                                                                                                                                            |
+| -t, \-\-tag        | One or more tags associated with topics and their respective number of partitions. For example:<br> `-t t01=partitions:3 -t t02=partitions:599999`.         |
```

Comment:
@davidcurrie I know this `tag` arg was added for internal use. Should we advertise this in the docs?

Reply:
Actually, the `tag` argument was not implemented at all. Instead, the approach was to allow an override file to be specified, which has the same format as the generated file. See the [PR description](https://github.com/corda/corda-runtime-os/pull/4857#issue-1940792518) for more details. I think it's fine to document the existence of the argument but I wouldn't go into any detail on its usage or include it in any of the examples.